app/scripts/views/mixins/flow-begin-mixin.js

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Tells the server to emit the `flow.begin` activity event.
+
+define(function (require, exports, module) {
+  'use strict';
+
+  var $ = require('jquery');
+
+  module.exports = {
+    afterRender: function () {
+      var flowId = this.user.get('flowId');
+      var flowBeginTime = $('body').attr('data-flow-begin');
+      this.metrics.logFlowBegin(flowId, flowBeginTime);
```

Comment:
should we check if `flowBeginTime` is set (or even send an error to sentry that is missing)? it can be missing if it is removed from the DOM or other weird reasons